scripts/screeps_rl_simulator_harness.py

```
@@ -815,6 +1577,51 @@ def run_self_test(stdout: TextIO = sys.stdout) -> int:
+        workers=workers,
+        room=room,
+        shard=shard,
+        branch=branch,
+        code_path=resolved_code_path,
+        map_source_file=resolved_map_source,
+        out_dir=resolved_out_dir,
+        run_id=resolved_run_id,
+    )
+    run_artifact_path = resolved_out_dir / resolved_run_id / "run_summary.json"
+    validate_run_artifact(artifact)
+    assert_no_secret_leak(artifact, dataset_export.configured_secret_values() + [os.environ.get("STEAM_KEY", "")])
```

Ruff (0.15.12)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)